hack/release.toml

```diff
@@ -31,6 +31,24 @@ Talos is built with Go 1.21.6.
         title = "Device Selectors"
         description = """\
 Talos Linux now supports `physical: true` qualifier for device selectors, it selects non-virtual network interfaces (i.e. `en0` is selected, while `bond0` is not).
+"""
+
+    [notes.extensions]
+        title = "Extension Services Config"
+        description = """\
+Talos now supports supplying configuration files for extension services that can be mounted into the extension service container.
+The extension service configuration is a separate config document. An example is shown below:
+
+```yaml
+---
+apiVersion: v1alpha1
+kind: ExtensionServicesConfig
+config:
```

Comment:
I feel like we have two standards here now. We use `spec` (and so does Kubernetes). We also don't use it in other places I can't remember at the moment. I feel like there should be some consistency across the formats.

Reply:
it's just a top level key and can be anything, for firewall rules it's something like this:

```yaml
apiVersion: v1alpha1
kind: NetworkDefaultActionConfig
ingress: block
---
apiVersion: v1alpha1
kind: NetworkRuleConfig
name: kubelet-ingress
portSelector:
  ports:
    - 10250
  protocol: tcp
ingress:
  - subnet: 172.20.0.0/24
    except: 172.20.0.1/32
```

we never used `spec` anywhere

Reply:
We do use `spec` ... `talosctl get` a resource and you can see. It feels inconsistent to me (and I feel like I represent an average user at this point :D)

Reply:
One of the goals this year is to clean up these types of things... resources names, resource layouts, etc.

Reply:
I don't want to block on this but I am going to work on making us all aware of this. I think this is a larger issue that needs to be tracked.